### ISSUE TITLE
Allow interactively picking the rotation center

### ISSUE THREAD

**Issue by greschd** [ansys/ansys-tools-visualization-interface#130]
### 📝 Description of the feature

When working with models whose dimensions are very unequal (e.g. a long tube-like model), it is important to be able to accurately set the rotation/zoom center.

Here's a video of a similar feature in Ansys Composite PrepPost (middle click changes rotation center [1]):

https://github.com/user-attachments/assets/19e9990f-7dab-4269-be25-2116849cde43

[1] it should be noted that middle click may not be the best hotkey, since it's often unavailable on Laptops. Maybe a "click a button - select new rotation center" flow would be more appropriate here

### Note

Before contributing and starting any work, please make sure you agree with the [CLA](https://developer.ansys.com/form/cla-acceptance) and sign it in your PR.

**Comment by ahernsean** [comment]
Don't force one particular UI methodology. Make setting the rotation center be an API call that users can then hook up as they wish.

**Comment by greschd** [comment]
Agreed; but I think we still ought to provide a sensible default experience.

**Comment by AlejandroFernandezLuces** [comment]
@pyansys-automation never close issue
